pandas/compat/_optional.py

```diff
@@ -16,7 +16,7 @@
     "odfpy": "1.3.0",
     "openpyxl": "2.4.8",
     "pandas_gbq": "0.8.0",
-    "pyarrow": "0.9.0",
+    "pyarrow": "0.12.0",
```

Comment:
@datapythonista here we need to update pytest

Reply:
Thanks for the clarification, I found it. It's already implemented in #29782